app/views/contexts/summary_context.py

```diff
@@ -24,7 +24,7 @@ def __call__(
 
     def _build_all_groups(
         self, return_to: Optional[str]
-    ) -> Generator[dict, None, None]:
+    ) -> Generator[Mapping[str, Any], None, None]:
```

Comment:
Return types should be concrete. So ImmutableDict/dict?

Reply:
👍